config/test.exs

```diff
@@ -14,11 +14,13 @@ config :trento, Trento.Repo,
   pool: Ecto.Adapters.SQL.Sandbox,
   pool_size: 10
 
-config :trento, Trento.Commanded,
-  event_store: [
-    adapter: Commanded.EventStore.Adapters.InMemory,
-    serializer: Commanded.Serialization.JsonSerializer
-  ]
+config :trento, Trento.EventStore,
```

Comment:
This gives us a real event store for test instead of an `InMemory` one.

I don't remember if we have tests needing an eventstore other than this one:
in that case those would now hit the database instead of using `InMemory` implementation.

I don't see that necessarily as a bad thing, I would just keep in mind that given the nature of the test, we might need a different implementation of the EventStore.

Reply:
This was configured, but we never used any EventStore in the tests.
We test our domain logic and handlers with unit tests/integration tests that do not pass from the EventStore.
The implementation of the rollup is specific to the Postgres event store, which is what we use, so I don't find particularly useful testing against the `InMeory` one.